linkerd/protocol/h2/src/main/scala/io/buoyant/linkerd/protocol/H2Config.scala

```diff
@@ -257,6 +257,7 @@ class H2ServerConfig extends ServerConfig with H2EndpointConfig {
   var maxConcurrentStreamsPerConnection: Option[Int] = None
   var addForwardedHeader: Option[AddForwardedHeaderConfig] = None
   var maxCallDepth: Option[Int] = None
+  var decoupleCloseAndGoAway: Option[Boolean] = Some(false)
```

Comment:
unused?

Reply:
Good catch. I was trying to make this configurable.